.github/workflows/ci-elasticsearch.yml

```diff
@@ -50,10 +44,24 @@ jobs:
       run: make install-ci
 
     - uses: docker/setup-qemu-action@68827325e0b33c7199eb31dd4e31fbe9023e06e3 # v3.0.0
+    
+    - name: Set up Elasticsearch
+      run: |
+           docker-compose -f  ./.github/workflows/docker-compose-elasticsearch-v${{ matrix.major-version }}.yml up -d
 
     - name: Run elasticsearch integration tests
       id: test-execution
-      run: bash scripts/es-integration-test.sh ${{ matrix.version.distribution }} ${{ matrix.version.image }}
+      run: | 
+          if [ "${{ matrix.major-version }}" = "7" ]; then
+            export ELASTICSEARCH_VERSION=$(grep 'image: docker.elastic.co/elasticsearch/elasticsearch' $(pwd)/.github/workflows/docker-compose-elasticsearch-v7.yml | cut -d ':' -f3)
```

Comment:
there's no point doing this if you simply pass only the major to `scripts/es-integration-test.sh` and let it start the appropriate docker-compose

Reply:
ok i have made the changes in scripts/es-integration-test.sh itself